bundles/org.openhab.binding.smaenergymeter/README.md

```diff
@@ -23,9 +23,21 @@ Optionally, a refresh interval (in seconds) can be defined.
 ## Channels
 
 -   **powerIn** Purchased power &lsqb;W&rsqb;
```

Comment:
Since this list has become much longer it's probably better to make this a markdown table (which is done in most other binding documentation)

```
| Channel | Description                   |
|---------|-------------------------------|
| powerIn | Purchased power &lsqb;W&rsqb; |
....
```
You can format the table with: http://markdowntable.com/

Reply:
I have created a md table as you suggested.

I left the phase mappings in the types for now, since, IMHO, they are not independent. But if anyone wants to change this across all files, I won't feel offended. 

For my taste, there are too many type dependencies spread all over the existing code, anyway. It is also not robust agains potential future changes, as it is using fixed offsets instead of OBIS IDs. But I wanted to submit a working delta before starting a major refactoring (if ever :) ).